Add interactive schema viewer references to migration documentation (#12558)
opensearch-project/documentation-website: _migration-assistant/workflow-cli/index.md

The CLI uses this schema automatically when editing and validating configurations, so no separate download is required.

> To browse all available fields, their types, defaults, and descriptions interactively, use the [Migration Assistant Schema Viewer](https://opensearch-project.github.io/opensearch-migrations/).

Reviewer: ```suggestion
For an interactive reference of all available fields, their types, defaults, and descriptions, see the [Migration Assistant Schema Viewer](https://opensearch-project.github.io/opensearch-migrations/).
```
